index.d.ts

```diff
@@ -1,18 +1,14 @@
-declare namespace inline {
-	interface Options {
-		attribute: string | boolean;
-		compress: boolean;
-		fs: any; // like MemoryFileSystem
-		handlers: [(source: any, context: any) => Promise<any>];
-		ignore: string[] | { [key: string]: string };
-		pretty: boolean;
-		rootpath: string;
-		saveRemote: boolean;
-		svgAsImage: boolean;
-		swallowErrors: boolean;
-	}
+export interface Options {
+	attribute: string | boolean;
+	compress: boolean;
+	fs: any; // like MemoryFileSystem
+	handlers: [(source: any, context: any) => Promise<any>];
+	ignore: string[] | { [key: string]: string };
+	pretty: boolean;
+	rootpath: string;
+	saveRemote: boolean;
+	svgAsImage: boolean;
+	swallowErrors: boolean;
 }
 
-declare function inline(htmlpath: string, options?: Partial<inline.Options>): Promise<string>;
-
-export = inline;
+export function inlineSource(htmlpath: string, options?: Partial<inline.Options>): Promise<string>;
```

Comment:
Shouldn't `inline.Options` be changed to `Options`?

Reply:
quite right.